src/accessibility-statement.md

```diff
@@ -1,198 +1,83 @@
 ---
-title: Accessibility statement for [website name]
+title: Accessibility statement for the GOV.UK Brand Guidelines website
 excludeFromNavigation: true
 ---
 
-<!--
-Note: you can have a single accessibility statement that covers multiple domains, or a separate statement for each domain or subdomain. As long as the user can access relevant accessibility information easily from any page on your website.
-
-For mobile apps also include the application’s version information and publication date.
-
-This wording is legally required and must not be changed.
--->
-
-This accessibility statement applies to \[scope of statement, for example, website or domain to which the statement applies\].
-
-<!--
-Note: use this section to make a brief, general statement about what the website allows disabled users to do. Base it on the evaluation covered in detail in the ‘Technical information about this website’s accessibility’ section.
-
-It must be specific to your own website. If you’re not confident that something is accurate, leave it out. If you’re not confident enough to say anything specific here, leave this section out completely.
--->
+This accessibility statement applies to govuk-brand-guidelines.service.gov.uk.
 
-This website is run by \[name of organisation\]. We want as many people as possible to be able to use this website. For example, that means you should be able to:
+This website is run by the Government Digital Service (GDS). We want as many people as possible to be able to use this website. For example, that means you should be able to:
 
 - change colours, contrast levels and fonts using browser or device settings
 - zoom in up to 400% without the text spilling off the screen
 - navigate most of the website using a keyboard or speech recognition software
 - listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
 
-We’ve also made the website text as simple as possible to understand.
-
-[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
+AbilityNet has advice on [making your device easier to use](https://mcmw.abilitynet.org.uk/) if you have a disability.
 
 ## How accessible this website is
 
-<!--
-Note: use this section to provide a summary of accessibility issues that a disabled user can act on \- for example, avoid a particular section of the website, or request an alternative version rather than waste time trying to make it work with their assistive technology. Try to list in order of most impact to least impact.
--->
-
 We know some parts of this website are not fully accessible:
 
-- ...
+- Some videos don't have a description and don't say they have no audio.
```

Comment:
This is where the most important issues go from the list below (under 'Non-compliance with the accessibility regulations'). I chose the two I think create the biggest barriers. But happy to add more.